en/controllers/components/authentication.rst

```diff
@@ -289,9 +289,13 @@ by including them in AuthComponents authenticate array::
 
     $this->Auth->config('authenticate', [
         'Openid', // app authentication object.
-        'AuthBag.Combo', // plugin authentication object.
+        'AuthBag.Openid', // plugin authentication object.
     ]);
 
+> Note that when using plugin notation there's no 'Authenticate' word when
```

Comment:
@antograssiot and @lorenzo thanks for the tips.  I'll do a commit ASAP making the required changes.